sdk/cosmosdb/cosmos/README.md

```diff
@@ -120,6 +121,34 @@ const { container } = await database.containers.createIfNotExists({ id: "Test Da
 console.log(container.id);
 ```
 
+### Using Partition Keys
+This example shows various types of partition Keys supported.
+```js
+await container.item("1").read();        // string type
+await container.item(2).read();          // number type
+await container.item(true).read();       // boolean type
+await container.item({}).read();         // None type
+await container.item(undefined).read();  // None type
```

Comment:
why do we have none type 2 times?

Reply:
Undefined and {} both map to none types.